### PR TITLE
Add some additional info to audiotest

### PR DESCRIPTION
**Pull request by forslund**
## Description
Audiotest now prints the device and samplerate used as well as the commandline used to playback the audio for easier debugging if things doesn't work as intended.

## How to test
Run the audiotest and check that the output matches the settings.

## Contributor license agreement signed?
CLA [ Yes ]

### CODE REVIEW

File: mycroft/util/audio_test.py [hunk 1]
```diff
@@ -80,6 +81,19 @@ def main():
         help="Add extra output regarding the recording")
     args = parser.parse_args()
 
+    config = Configuration.get()
+    if "device_name" in config["listener"]:
+        dev = config["listener"]["device_name"]
+    elif "device_index" in config["listenter"]:
```
Comment (penrods) [inline mycroft/util/audio_test.py]: Typo: listenter

File: mycroft/util/audio_test.py [hunk 2]
```diff
@@ -80,6 +81,19 @@ def main():
...
+        dev = "Device at index {}".format(config["listener"]["device_index"])
+    else:
+        dev = "Default device"
+    samplerate = config["listener"]["sample_rate"]
+    play_cmd = config["play_wav_cmdline"].replace("%1", "WAV_FILE")
+    print(" ============================= Info ==============================")
```
Comment (penrods) [inline mycroft/util/audio_test.py]: Nice!  Would also be nice to list all devices when someone is looking for a mic.  I'm going to add a -l option.

Comment (forslund) [reply mycroft/util/audio_test.py]: Sounds good

File: mycroft/util/audio_test.py (hunk 1 above)
Comment (forslund) [reply mycroft/util/audio_test.py]: Nice catch

**Comment by forslund** [comment]
Nice addition. Merging

**Pull request merged**